src/EdFi.AnalyticsMiddleTier.DataStandard31/Asmt/MSSQL/0011-View-AssessmentFact-Create.sql

```diff
@@ -0,0 +1,76 @@
+
+IF EXISTS (SELECT 1 FROM INFORMATION_SCHEMA.VIEWS WHERE TABLE_SCHEMA = 'analytics' AND TABLE_NAME = 'asmt_AssessmentFact')
+BEGIN
+	DROP VIEW analytics.asmt_AssessmentFact
+END
+GO
+
+CREATE VIEW analytics.asmt_AssessmentFact
+AS
+	SELECT
+       CONCAT(Assessment.AssessmentIdentifier, '-', Assessment.Namespace)  AS AssessmentKey,
+       Assessment.Namespace,
+       Assessment.AssessmentTitle AS Title,
+       COALESCE(Assessment.AssessmentVersion, 0) AS Version,
+       COALESCE(CategoryDescriptor.Description, '') AS Category,
+       Descriptor.Description AS AssessedGradeLevel,
+       COALESCE(AcademicSubjectDescriptor.Description,'') AS AcademicSubject,
+       CONCAT(ObjectiveAssessment.AssessmentIdentifier, '-', ObjectiveAssessment.IdentificationCode, '-', ObjectiveAssessment.Namespace) AS ObjectiveAssessmentKey,
+       ObjectiveAssessment.IdentificationCode,
+       CASE WHEN ObjectiveAssessment.ParentIdentificationCode IS NOT NULL
+            THEN CONCAT(ParentAssesment.AssessmentIdentifier, '-', ParentAssesment.IdentificationCode, '-', ParentAssesment.Namespace)
+          ELSE ''
+          END AS ParentObjectiveAssessmentKey,
+       COALESCE(ObjectiveAssessment.Description, '') as ObjectiveAssessmentDescription,
+       COALESCE(ObjectiveAssessment.PercentOfAssessment,0) as PercentOfAssessment,
+       COALESCE(AssessmentScore.MinimumScore, ObjectiveAssessmentScore.MinimumScore, '') as MinScore,
+       COALESCE(AssessmentScore.MaximumScore, ObjectiveAssessmentScore.MaximumScore, '') as MaxScore
+    FROM 
+         edfi.Assessment
+    INNER JOIN
+        edfi.AssessmentAssessedGradeLevel ON
+            Assessment.AssessmentIdentifier = AssessmentAssessedGradeLevel.AssessmentIdentifier
+            AND
+            Assessment.Namespace = AssessmentAssessedGradeLevel.Namespace
+    INNER JOIN
+        edfi.AssessmentScore ON
+            Assessment.AssessmentIdentifier = AssessmentScore.AssessmentIdentifier
+            AND
+            Assessment.Namespace = AssessmentScore.Namespace
+    INNER JOIN
+        edfi.Descriptor ON
+            Descriptor.DescriptorId = AssessmentAssessedGradeLevel.GradeLevelDescriptorId
+    LEFT JOIN
+        edfi.AssessmentAcademicSubject ON
+           Assessment.AssessmentIdentifier = AssessmentAcademicSubject.AssessmentIdentifier
+           AND
+           Assessment.Namespace = AssessmentAcademicSubject.Namespace
+    LEFT JOIN
+        edfi.Descriptor AcademicSubjectDescriptor ON
+            AcademicSubjectDescriptor.DescriptorId = AssessmentAcademicSubject.AcademicSubjectDescriptorId
+    LEFT JOIN
+        edfi.Descriptor CategoryDescriptor ON
+            CategoryDescriptor.DescriptorId = Assessment.AssessmentCategoryDescriptorId
+    LEFT JOIN
+        edfi.ObjectiveAssessment ON
+            Assessment.AssessmentIdentifier = ObjectiveAssessment.AssessmentIdentifier
+            AND
+            Assessment.Namespace = ObjectiveAssessment.Namespace
```

Comment:
Missing test cases to validate the namespace if this condition is removed, no test fails.